common/materialContainer/MaterialContainer.h

```diff
@@ -76,7 +76,7 @@ template <class T> class MaterialContainer
     , contentFreq(MC.contentFreq)
   {
   }
-  MaterialContainer& operator=(const MaterialContainer& mc) { return *this; }
+  MaterialContainer& operator=(const MaterialContainer&) { return *this; }
```

Comment:
I don't understand the need for a copy method on this object. If there is a valid reason, then I believe assigning all the fields is the accurate route.